pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -518,6 +522,25 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 			return reconcile.Result{}, err
 		}
 
+		customSnortRulesCm := new(corev1.ConfigMap)
```

Comment:
ConfigMaps can store up to 1 MiB of data, while the set of snort rules may quite possibly be larger than this.